scripts/quests/otherAreas/An_Explorers_Footsteps.lua

```diff
@@ -0,0 +1,358 @@
+-----------------------------------
+-- An Explorer's Footsteps
+-----------------------------------
+-- Log ID: 4, Quest ID: 19
+-- This quest was changed to require a minimum amount of fame to combat RMTs POS-Hacking around to
+-- quickly earn gil. However, as this is not a legitimate concern on private servers players may
+-- complete this quest even with no fame.
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.AN_EXPLORER_S_FOOTSTEPS)
+
+local payOut =
+{
+    800, -- West Ronfaure -- !gotoid 17187513
+    800, -- East Ronfaure -- !gotoid 17191526
+    1000, -- La Theine Plateau -- !gotoid 17195621
+    1000, -- Valkurm Dunes -- !gotoid 17199698
+    1000, -- Jugner Forest -- !gotoid 17203814
+    10000, -- Batallia Downs -- !gotoid 17207821
+    3000, -- North Gustaberg -- !gotoid 17212049
+    800, -- South Gustaberg -- !gotoid 17216174
+    1000, -- Konschtat Highlands -- !gotoid 17220150
+    1000, -- Pashhow Marshlands -- !gotoid 17224302
+    3000, -- Rolanberry Fields -- !gotoid 17228353
+    800, -- West Sarutabaruta -- !gotoid 17248818
+    800, -- East Sarutabaruta -- !gotoid 17253064
+    1000, -- Tahrongi Canyon -- !gotoid 17257053
+    1000, -- Buburimu Peninsula -- !gotoid 17261140
+    1000, -- Meriphataud Mountains -- !gotoid 17265242
+    10000, -- Sauromugue Champaign -- !gotoid 17269222
```

Comment:
If a server is set up as multi-process, gotoid won't work here. These could/should be !pos with x, y, z, zone, etc.

Reply:
Will update soon